src/main/java/com/keybox/manage/action/UsersAction.java

```diff
@@ -121,17 +121,21 @@ public void validateSaveUser() {
                 || user.getFirstNm().trim().equals("")) {
             addFieldError("user.firstNm", REQUIRED);
         }
-        
-        if (user != null && user.getPassword() != null && !user.getPassword().trim().equals("")){
-            
-            if(!user.getPassword().equals(user.getPasswordConfirm())) {
+
+        // Password validation is only for users who are permitted to log in (ie non-passive ones)
+        if (user != null && !User.PASSIVE.equals(user.getUserType())) {
+            if (user != null && user.getPassword() != null && !user.getPassword().trim().equals("")){
```

Comment:
Removing this wrap would mean therefore that you'd need to supply a password when creating one of these users? That seems counter-intuitive - I can update the duplicate null check and then adjust the setting-password thing accordingly if that works instead.